packages/riipen-ui/src/components/Tooltip.jsx

```diff
@@ -0,0 +1,429 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+import css from "styled-jsx/css";
+
+import ThemeContext from "../styles/ThemeContext";
+
+import withClasses from "../utils/withClasses";
+
+import Popover from "./Popover";
+
+class Tooltip extends React.Component {
+  static propTypes = {
+    /**
+     * The tooltip content.
```

Comment:
I would maybe explain this one a bit more, it's a bit confusing given the documentation for the `tooltip` prop. I would maybe say "The content to trigger the tooltip with". or something.